File: unittest/scripts/py_dev_api_examples/concepts/Connecting_to_a_Single_MySQL_Server.py
# Passing the parameters in the { param: value } format
dictSession = mysqlx.get_session( {
        'host': 'localhost', 'port': 33060,
        'user': 'mike', 'password': 'paSSw0rd' } )

db1 = dictSession.get_schema('test')

# Passing the parameters in the URL format
uriSession = mysqlx.get_session('mike:paSSw0rd@localhost:33060')

db2 = uriSession.get_schema('test')

File: unittest/scripts/js_dev_api_examples/concepts/Connecting_to_a_Single_MySQL_Server.js
// Passing the parameters in the { param: value } format
var dictSession = mysqlx.getSession( {
        host: 'localhost', 'port': 33060,
        user: 'mike', password: 'paSSw0rd' } )

var db1 = dictSession.getSchema('test')

// Passing the parameters in the URL format
var uriSession = mysqlx.getSession('mike:paSSw0rd@localhost:33060')

var db2 = uriSession.getSchema('test')
